README.md

```diff
@@ -106,6 +106,7 @@ BuildKit is used by the following projects:
 -   [Dagger](https://dagger.io)
 -   [envd](https://github.com/tensorchord/envd/)
 -   [Depot](https://depot.dev)
+-   [Namespace](https://namespace.so)
```

Comment:
Would linking to https://github.com/namespacelabs/foundation be alright?

Reply:
If possible I'd keep namespace.so because that better reflects our broader use of buildkit, and is akin to some of the other URLs above that refer to the product (vs the implementation). But I'll take your guidance here if you feel a pointer to a repo is more appropriate! Thanks!